### ISSUE TITLE
Overview of work - Main Menu Reformatting

### ISSUE THREAD

**Issue by sciencebarge** [nicholasericksen/GHV-data#22]
Hi all.  After a YEAR of putting this aside, we are full speed ahead on some big changes and wrapping up the work before we unveil our distance learning module.

In the next few weeks, we will be reorganizing content to fit into three homepage tabs/links: 
- **Climate Science**
        (most of the content currently under "Climate Monitoring" will remain under this heading)
- **Climate Change Impact**
        (this will now encompass all the content from "Impact to Biotic Communities" and "Impact to Water 
        Cycle & Resources" as well as a few new resources)
- **Adaptation and Resiliency**
       (this will have all new content, including either video tours of each site, or VR experiences)

In addition, I'm in the process of buying new sizable button icons to possibly change the aesthetic of the main menus, so their is a visual representation to go along with the button text.  I'll include those in a different thread, though they are not a priority as much as fixing and updating content is.

We will be removing some dead links altogether.  I'll start a CLEAR OUT YOUR DEAD thread for that.

We may repair some links or swap sources.  I'll start a WE CAN REBUILD HIM: WE HAVE THE TECHNOLOGY thread for that.

We will definitely be adding content.  I'll start a "TRY NOT: DO OR DO NOT, THERE IS NO TRY" thread for that.

In the meantime.... can we try to find ways to display as many images in full screen?  I can identify specific ones, but especially graphs that need to be used.... Full screen or close to it is best.

**Comment by sciencebarge** [comment]
Once our new layout is in place, I'll have more captions. 

In the meantime, I was wondering if we can **restructure captions to be in a bar on the left-hand side** instead of underneath.  I think that will help to increase the size of some images that are appearing too small to read in many cases, and help with readability of the caption as well.

Overall, I'd like many of the images to display larger if possible, too.

**Comment by nicholasericksen** [comment]
Everything should have been taken care of
